crowdin_updater.py

```diff
@@ -98,7 +98,7 @@ def upload(self, inputfile, parentdir=None):
     # 1 if success
     # 0 if unknown status
     def parseXMLResponse(self, r):
-        if r.headers['content-type'] != 'text/xml':
+        if r.headers['content-type'].startswith('text/xml'):
```

Comment:
Would it be better to use "contains" instead of "starts with"?

```py
if 'text/html' in r.headers['content-type']:
```

Reply:
No need, a Content-Type header will always starts with the media type (RFC 2045 §5.1)

Reply:
According to the spec, the `Content-Type` should start with `type/subtype`, so let's check that.